modules/beamdyn/src/BeamDyn.f90

```diff
@@ -2418,6 +2414,8 @@ SUBROUTINE BD_RotationalInterpQP( nelem, p, x, m )
    TYPE(BD_ContinuousStateType), INTENT(IN   )  :: x                 !< Continuous states at t
    TYPE(BD_MiscVarType),         INTENT(INOUT)  :: m                 !< misc/optimization variables
 
+   INTEGER(IntKi)                :: ErrStat           !< index to current element
```

Comment:
copy paste error on comment.

Since we have an ErrStat now, it should probably be returned up through the call stack rather than ignored.